tests/test_integrated_channels/test_blackboard/test_transmitters/test_content_metadata.py

```diff
@@ -66,15 +66,28 @@ def test_transmit_content_metadata_updates_records(
             integrated_channel_code=self.enterprise_config.channel_code(),
             content_last_changed='2021-07-16T15:11:10.521611Z',
             enterprise_customer_catalog_uuid=self.enterprise_customer_catalog.uuid,
-            channel_metadata={}
+            channel_metadata={},
```

Comment:
this same test pattern is copied to almost every channel implementation. we have an opportunity here to DRY these tests into a single base test that takes all the implementations as input.